recipe/meta.yaml

```diff
@@ -36,28 +36,31 @@ requirements:
     ### main packages ###
     - python
     - r
-    - cime_gen_domain  # [linux]
+    - cdo
+    - cime_gen_domain 6.1.59
     - chemdyg 1.0.1
-    - e3sm_diags 2.12.0
-    - e3sm_to_cmip 1.11.2
-    - e3sm-tools 3.0.0  # [linux]
-    - geometric_features 1.3.0
+    - e3sm_diags 2.12.1  # 3.0.0rc1
```

Comment:
The release candidate isn't there yet so we're just testing things out while we wait.